similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/AbstractPatientGenotypeSimilarityView.java

```diff
@@ -93,7 +93,7 @@ public boolean hasGenotypeData()
             Set<String> matchGenes = this.matchGenotype.getCandidateGenes();
             Set<String> refGenes = this.refGenotype.getCandidateGenes();
             sharedGenes.addAll(matchGenes);
-            sharedGenes.retainAll(refGenes);
+            sharedGenes.addAll(refGenes);
```

Comment:
If the meaning of the method changes, then its name should change as well, but that would be an API breakage.